CHANGES.rst

```diff
@@ -12,7 +12,9 @@ Changes
 
 Changes:
 --------
-- No change.
+- Fix `http` directory download to match implemented `AWS S3` directory download in `weaver.utils.fetch_directory`,
```

Comment:
add double quotes for `http` and `weaver.utils.fetch_directory`

Reply:
Revert the double quotes for ``` `AWS S3` ```. See other examples in the file. Double quotes for "code stuff" and single ones for italic emphasis on concepts.